<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLite Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="cntrSideBar"/>
   </widget>
   <widget class="QFrame" name="newTableFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>481</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="newTableGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>531</height>
      </rect>
     </property>
     <property name="title">
      <string>New Table</string>
     </property>
     <widget class="QLabel" name="newTableNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Table Name</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="newTableNameText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="ifNotExistsCheckBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>If Not Exists</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="replaceExistingCheckBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Or Replace</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTableFieldButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Field</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="newTableFieldGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>461</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>New Field</string>
      </property>
      <widget class="QLabel" name="newFieldNameLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>New Field Name</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="newFieldNameText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>200</width>
         <height>25</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QLabel" name="newFieldTypeLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>100</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Field Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="newFieldTypeComboBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>130</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="isPrimaryKeyCheckBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Primary Key</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="isNullableCheckBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nullable</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="newFieldButtonBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>241</width>
         <height>24</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="isFieldUniqueCheckBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Unique</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="newTableSqlPreviewLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>461</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QDialogButtonBox" name="newTableOkCancel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>156</width>
       <height>24</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="keepNewTableEntryCheckBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>3</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Retain on Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionNewTable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewTable">
   <property name="text">
    <string>New Table</string>
   </property>
   <property name="toolTip">
    <string>Open new table creator.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
